spec/factories/report.rb

```diff
@@ -13,5 +13,13 @@
       state { :approved }
       deadline { 1.day.from_now }
     end
+
+    trait :ispf do
+      association :fund, :ispf, factory: :fund_activity
+    end
+
+    trait :gcrf do
+      association :fund, :gcrf, factory: :fund_activity
+    end
```

Comment:
Not sure about this, but I wonder if it would be more in line with the activity factory to call these e.g. `ispf_funded` or `ispf_associated`? The bare `ispf` is only used when creating the ISPF fund activity itself in the activity factory, whereas at lower levels where it's an association, it's more adjectival(?)

Reply:
We're reporting against a specific fund, so `ispf_funded` or `_associated` doesn't feel quite right to me. I'd say it's an "ISPF report" - is there a clearer way we might be able to convey that in the trait's name?

Reply:
Not sure I understand - is `ispf_associated` not specific to the ISPF fund? Maybe "associated" it makes the connection sound too weak/optional?

My point here is the same as the one below - the report isn't ISPF itself, but it's an ISPF report as you say. I guess I'm seeing traits as something that you could put in a sentence as "it is [trait]", so for activities "it is ISPF" works for funds while "it is ISPF-funded" works at lower levels, and here a report isn't ISPF itself but it's for/against ISPF. This understanding of trait naming may be a result of most of my exposure to traits coming from the RODA activity factory though - no idea if it's a wider convention!

This is very minor in any case!

Reply:
Have gone for `for_ispf` as it felt more concrete than "associated", which felt a bit optional/loosely related